frontend/benefit/applicant/src/components/applications/pageContent/PageContent.tsx

```diff
@@ -12,23 +14,37 @@ import ApplicationFormStep6 from 'benefit/applicant/components/applications/form
 import React from 'react';
 import Container from 'shared/components/container/Container';
 import Stepper from 'shared/components/stepper/Stepper';
+import { formatDate } from 'shared/utils/date.utils';
 
 import { usePageContent } from './usePageContent';
 
 const PageContent: React.FC = () => {
-  const { t, steps, currentStep, application } = usePageContent();
+  const { t, id, steps, currentStep, application } = usePageContent();
   return (
     <Container>
       <$PageHeader>
         <$HeaderItem>
           <$PageHeading>
-            {t('common:applications.pageHeaders.new')}
+            {t(`common:applications.pageHeaders.${id ? 'edit' : 'new'}`)}
           </$PageHeading>
         </$HeaderItem>
         <$HeaderItem>
           <Stepper steps={steps} activeStep={currentStep} />
         </$HeaderItem>
       </$PageHeader>
+      {id && application?.createdAt && (
+        <>
+          <$PageSubHeading>
+            {`${t('common:applications.pageHeaders.created')} ${formatDate(
+              new Date(application?.createdAt),
+              'dd.MM.yyyy. mm:ss'
```

Comment:
I noticed that this format is used in many places, I suggest extracting it as a constant.

Reply:
yes, it is reusable. i ll fix